### PR TITLE
fix: Judge email address should be shown only to HMCTS Users(Judges, Admin and Gatekeepers)[FPLA-1773]

### PR DESCRIPTION
**Pull request by mounikahmcts**
[FPLA-1773](https://tools.hmcts.net/jira/browse/FPLA-1773)

Judge'e email addresses in the tabbed view only for HMCTS users 
https://tools.hmcts.net/confluence/display/RCCD/CRUD+on+Complex+Types

### CODE REVIEW

File: e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ Scenario('Judiciary adds allocated judge', async (I, caseViewPage, allocatedJudg
   caseViewPage.selectTab(caseViewPage.tabs.casePeople);
   I.seeInTab(['Allocated Judge', 'Judge or magistrate\'s title'], 'Her Honour Judge');
   I.seeInTab(['Allocated Judge', 'Last name'], 'Moley');
+  I.seeInTab(['Allocated Judge', 'Email Address'], 'moley@example.com');
```
Comment (swalker125) [inline e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js:36]: I think I expect to see somewhere in e2e that user cannot see judge email address

Comment (mounikahmcts) [reply e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js:36]: Added test for LA - not seeing judge email address.

Comment (mounikahmcts) [reply e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js:36]: > I think I expect to see somewhere in e2e that user cannot see judge email address

Updated tests with LA user who cannot see the judge email

File: e2e/steps_file.js [hunk 2]
```diff
@@ -103,6 +107,10 @@ module.exports = function () {
       }
     },
 
+    dontSeeInTab(pathToField) {
+      this.dontSeeElement(locate(this.tabFieldSelector(pathToField)));
+    },
```
Comment (Jamesnelson117) [inline e2e/steps_file.js:112]: 👍

File: e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js [hunk 3]
```diff
@@ -33,6 +33,16 @@ Scenario('Judiciary adds allocated judge', async (I, caseViewPage, allocatedJudg
...
+
+  await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  I.seeInTab(['Allocated Judge', 'Judge or magistrate\'s title'], 'Her Honour Judge');
+  I.seeInTab(['Allocated Judge', 'Last name'], 'Moley');
+  I.dontSeeInTab(['Allocated Judge', 'Email Address']);
```
Comment (Jamesnelson117) [inline e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js:43]: Minor but is the email address suppose to be title case?

File: ccd-definition/AuthorisationComplexType.json [hunk 4]
```diff
@@ -1 +1,194 @@
-[]
+[
+    {
```
Comment (Jamesnelson117) [inline ccd-definition/AuthorisationComplexType.json:2]: Minor spacing issue

File: e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js (hunk 3 above)
Comment (mounikahmcts) [reply e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js:43]: > Minor but is the email address suppose to be title case?

https://tools.hmcts.net/jira/browse/FPLA-1664 - it was developed as part of this story and is approved. Do you think i should change it?

Comment (Jamesnelson117) [reply e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js:43]: As discussed maybe out of scope for this story :(

**Pull request merged**